lib/gltfPipeline.js

```diff
@@ -52,6 +53,13 @@ function writeSources(gltf) {
         });
 }
 
+/**
+ * Add pipeline extras and load uris, then process the gltf.
+ *
+ * @param {Object} gltf A javascript object holding a glTF hierarchy.
+ * @param {Object} options Options to apply to stages during optimization.
```

Comment:
We pass options to stages and they should be documented for all of those. I think it would be a bit verbose to document all of the different stage options here.